BO4E-dotnet/COM/Verbauchsmenge.cs

```diff
@@ -0,0 +1,15 @@
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+
+namespace BO4E.COM
+{
+    /// <summary>Verbauchsmenge von Marktlokation</summary>
+    public class Verbauchsmenge : Menge
```

Comment:
Ich glaube, es sollte "Verb**r**auchsmenge" heißen. Bitte auch in den Docstrings anpassen.

Reply:
Ach Flo hat Falsch in Excel geschrieben :D